kubernetes/chart/templates/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+# This is a simple example of using a config map to create a single page static site.
```

Comment:
According to the instructions the kubectl apply will include the ConfigMap, what about upgrading? do we need to store our configmap data in here as well? Or just provide instructions on building a configmap as a separate step?

Reply:
I'm on the fence about this. The configmap isn't hard to add to the chart itself, and for ease of installing.  Upgrading would require to regenerate the template anywho with an updated values.yaml.

Reply:
Right now the config seems like the weak point as it would be nice to read directly from vault.  I'm going to look to see if we can use vault directly with kubernetes.